Dockerfile.backwards

```diff
@@ -7,6 +7,8 @@ RUN echo "yaml https://s3-us-west-2.amazonaws.com/rosdep/base.yaml" > /etc/ros/r
 COPY . /opt/package
 WORKDIR /opt/package
 
+RUN apt update || apt install -y curl && curl -s https://raw.githubusercontent.com/ros/rosdistro/master/ros.asc | apt-key add -
```

Comment:
Use `apt-get` instead of `apt`.
See: https://manpages.ubuntu.com/manpages/xenial/man8/apt.8.html#script%20usage%20and%20differences%20from%20other%20apt%20tools